tests/views_integration/Tribe/Events/Views/V2/Partials/Components/Top_Bar/TodayTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Tribe\Events\Views\V2\Partials\Components\Top_Bar;
+
+use Tribe\Test\Products\WPBrowser\Views\V2\HtmlPartialTestCase;
+
+class TodayTest extends HtmlPartialTestCase
+{
+
+	protected $partial_path = 'components/top-bar/today';
+
+	/**
+	 * Test render with context
+	 */
+	public function test_render_with_context() {
+		$this->assertMatchesSnapshot( $this->get_partial_html( [
+			'today_url' => '#',
```

Comment:
I'd rather user hard-coded values here, this would be easier to spot in the snapshot if something changes.